asv.conf.json

```diff
@@ -13,6 +13,9 @@
     // project being benchmarked
     "repo": ".",
 
+    // Install using default qt install
+    "install_command": ["in-dir={env_dir} python -mpip install {conf_dir}[all]"],
```

Comment:
I had to add this to specify the qt option. I chose `[all]`, but I don't have a strong preference which backend we use.

This assumes that the config file (`asv.conf.json`) lives in the root directory where the installation instructions are. I think this is a safe assumption, but it would be nice to get some feedback if this is the best way to do this.

Reply:
Also, I modified this from this [asv docs](https://asv.readthedocs.io/en/stable/asv.conf.json.html?highlight=asv%20conf#build-command-install-command-uninstall-command). Does anybody know why there isn't a space between `python -m` and `pip`? It works, but it seems odd to me.

Reply:
@jni do you have thoughts here? Given that is from the docs I am fine with it

Reply:
You can add a space if it makes you feel better. It would make *me* feel better too. =P But the GNU argument syntax is flexible, and if your argument has short form `a` and long form `option`, all of the following are valid:

- `-a argvalue`
- `-aargvalue`
- `--option argvalue`
- `--option=argvalue`

Reply:
Ah TIL. Thanks for the explanation!